corehq/apps/data_interfaces/views.py

```diff
@@ -909,13 +863,50 @@ def initial_rule(self):
         return rule
 
 
-class DeduplicationRuleListView(AutomaticUpdateRuleListView):
+class DeduplicationRuleListView(DataInterfaceSection, CRUDPaginatedViewMixin):
     template_name = 'data_interfaces/list_deduplication_rules.html'
     urlname = 'deduplication_rules'
     page_title = gettext_lazy("Deduplicate Cases")
 
+    ACTION_ACTIVATE = 'activate'
+    ACTION_DEACTIVATE = 'deactivate'
+
     rule_workflow = AutomaticUpdateRule.WORKFLOW_DEDUPLICATE
 
+    def post(self, *args, **kwargs):
+        return self.paginate_crud_response
+
+    @property
+    def page_context(self):
+        context = self.pagination_context
+        domain_obj = Domain.get_by_name(self.domain)
+        hour = domain_obj.auto_case_update_hour
+        context.update({
+            'help_site_url': 'https://confluence.dimagi.com/display/commcarepublic/Automatically+Close+Cases',
+            'time': f"{hour}:00" if hour else _('midnight'),
+        })
+        return context
+
+    @property
+    def paginated_list(self):
+        for rule in self._rules()[self.skip:self.skip + self.limit]:
+            yield {
+                'itemData': self._format_rule(rule),
+                'template': 'base-rule-template',
+            }
+
+    @property
+    def total(self):
+        return self._rules().count()
+
+    @memoized
+    def _rules(self):
+        return AutomaticUpdateRule.by_domain(
+            self.domain,
+            self.rule_workflow,
+            active_only=False,
+        ).order_by('name', 'id')
```

Comment:
Nitpick, but you handled it so nicely on the js side: https://code.djangoproject.com/ticket/6498

Reply:
Offlined with Jenny about this, but after a bit of a dive, I learned that the django ORM does not override however the backing database handles `ORDER BY` in SQL. In our case, it appears that both `LC_COLLATE` or `LC_CTYPE` is `en_US.utf8` which apparently treats upper case equivalent to lowercase. I compared this to a different type in a raw sql query and saw results sorted in case sensitive order. Based on [this code](https://github.com/dimagi/commcare-cloud/blob/ea3086963827e7beae59252667ca7223732b0f3c/src/commcare_cloud/ansible/roles/postgresql_base/tasks/set_up_dbs.yml#L96-L97) in commcare-cloud, all of our db instances should respect this.

Reply:
thank you for tracking that down, it's good to know